index.ts

```diff
@@ -21,29 +21,28 @@ export function* sinergia(
       const itemIterator: IterableIterator<any> = task(accumulator, item);
 
       yield new Promise(resolve => {
-        const step = (timestamp) => { // timestamp not used
+        const step = () => {
           const iteration = itemIterator.next();
 
-          if (!iteration.done) {
-            window.requestAnimationFrame(step);
-          }
-          else {
+          if (iteration.done) {
             if (actualOptions.debug) console.log(`item task is done with latest iteration`, iteration);
             accumulator = iteration.value;
-            window.cancelAnimationFrame(animToken);
             resolve();
           }
+          else {
+            animToken = requestAnimationFrame(step);
+          }
         };
 
-        animToken = window.requestAnimationFrame(step);
+        animToken = requestAnimationFrame(step);
       });
     }
 
     return { value: accumulator };
   } finally {
     // This block is called when sinergia is interrupted with `.return()`
 
-    if (animToken) window.cancelAnimationFrame(animToken);
```

Comment:
I mean which can be ran both in browser and Node. The package is browser-only but still there are benifits in making explicit browser-only functions:

1. It makes it clear for anyone seeing the source code that this lib is not meant for Node.
2. When writing unit tests you'll know which functions need to be mocked because I'm using Jest and not Karma, so they run in Node without jsdom or electron